model-api/src/commonMain/kotlin/org/modelix/model/api/INode.kt

```diff
@@ -324,7 +324,11 @@ fun INode.resolveProperty(role: String): IProperty {
 }
 
 fun INode.tryResolveChildLink(role: String): IChildLink? {
-    val c = this.concept ?: return null
+    val c = try {
+        this.concept ?: return null
+    } catch (e: RuntimeException) {
+        return null
```

Comment:
This doesn't look like a proper fix. It just makes it harder to debug if there is a missing language. The relation between MODELIX-714 and this change is also not clear, so it's hard to propose an alternative solution.

Reply:
To avoid such design discussion in the future, it would probably be good to specify what the `try` in the method names of the patched methods is intended to do. With this patch, `try` is extended to include the cases where a concept is unknown.